packages/fluentui/accessibility/src/behaviors/Box/dropdownClearIndicatorBehavior.ts

```diff
@@ -0,0 +1,24 @@
+import { Accessibility } from '../../types';
+
+/**
+ * @description
+ * Indicator is usually only visual representation and therefore is hidden from screen readers, unless 'aria-label' property is provided.
+ * Adds attribute 'tabIndex=-1' to 'root' slot if 'search' property is false. Does not set the attribute otherwise.
```

Comment:
```suggestion
 * Adds attribute 'tabIndex=0' to 'root' slot if 'search' property is false. Does not set the attribute otherwise.
```

Reply:
And shouldn't it be in the `@specification` part?

Reply:
Thank you!